apps/fabric-website/src/SiteDefinition/SiteDefinition.pages/Controls/cross.tsx

```diff
@@ -18,11 +18,55 @@ export const controlsPagesCrossPlatform: INavPage[] = [
     pages: [
       {
         title: 'Button',
-        url: '#/controls/crossplatform/button',
+        url: '#/controls/cross/button',
```

Comment:
so the plan is to use /cross for the crossplatform path? was that very intentional?

Reply:
I wonder if `reactnative` would be better? Saying this having not been fully following all your work, so feel free to ignore if that doesn't make sense. :)

Reply:
I added back the "crossplatform" route. Elizabeth, you wouldn't believe the back and forth that went on around the naming of this section, but we're sticking with Cross-platform for now :).

Reply:
Eh I'm not too surprised... 😆